doc/Language/objects.pod6

```diff
@@ -263,13 +260,13 @@ Method names can be resolved at runtime with the C<.""> operator.
 =head2 self
 
 Inside a method, the term C<self> is available, which is bound to the
-invocant, i.e.  the object that the method was called on. C<self> can be
+invocant, the object that the method was called on. C<self> can be
 used to call further methods on the invocant. Within methods, something like
-C<$.origin> is the same thing as C<self.origin>, however the colon-syntax for
+C<$.origin> is the same thing as C<self.origin>; however, the colon-syntax for
```

Comment:
This was better before. It would be great if the sentence didn't use the words “something like” or “same thing”.